### PR TITLE
Cleanup: removed typo, updated readme, deleted unimportant files

### PR DESCRIPTION
**Pull request by Lukuas31415**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,33 @@
-# BA
-My Bachelorthesis Repository
+# A cuboid muscle model
 
-**Content:**
+## Setup
+- A dummy cuboid muscle geometry. 
+- Solver comprises hyperelasticity solver for the prestretch and both mechanics solver + fastmonodomain solver for the contraction.
```
Comment (carme-hp) [inline README.md:5]: no longer true, it is a FastMonodomainSolver + MuscleContractionSolver. See the [readme](https://github.com/opendihu/opendihu/blob/95fbdfe31be284697846d218570726ac544b27ec/examples/electrophysiology/neuromuscular/cuboid_muscle_with_prestretch/README.md) I wrote for the opendihu repo

File: README.md [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+mkorn && sr
+```
+For a debug build, look into the documentation. 
 
--BayesOpt
+### How to run?
+To run a single simulation of stretching a muscle with a certain force and then contract it, go to build_release and run:
+
+```
+./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py incompressible_mooney_rivlin 10.0
```
Comment (carme-hp) [inline README.md:20]: you can directly run it with 
`./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py 10.0`

**Review by carme-hp — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Lukuas31415) [reply README.md:20]: But then you can't specify the force. Line 28 in settings_contraction_with_prestretch.py requires you to add a scenario name, else you can't hand over the force value.

File: README.md (hunk 1 above)
Comment (Lukuas31415) [reply README.md]: Yes, I have added this now.

**Review by carme-hp — APPROVED** [review]

**Comment by carme-hp** [comment]
Changes look good, you can merge:)

**Pull request merged**